docs/examples/containerized_heketi_dedicated_gluster/README.md

```diff
@@ -0,0 +1,669 @@
+  selector:
+    name: heketi
+  ports:
+  - name: heketi
+    port: 8888
+    targetPort: 8888
+
+```
+
+heketi-rc.yaml
+```
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: heketi
+spec:
+  replicas: 1
+  selector:
+    app: heketi
+  template:
+    metadata:
+      name: heketi
+      labels:
+        app: heketi
+    spec:
+      hostNetwork: true
+      containers:
+      - name: heketi
+        image: heketi/heketi:dev
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8888  <1>
+        volumeMounts:
+        - name: config  <2>
+          mountPath: /etc/heketi
+        - name: db  <3>
+          mountPath: /var/lib/heketi
+        - name: keys  <4>
+          mountPath: /usr/share/keys
+      volumes:
+        - name: keys  <4>
+          hostPath:
+            path: /usr/share/heketi/keys
+        - name: config  <2>
+          hostPath:
+            path: /usr/share/heketi
+        - name: db  <3>
+          glusterfs:
+            endpoints: glusterfs-cluster
+            path: MyHeketi
+```
+<1>  Match container port to service and heketi.json.
+
+<2>  Set up a Heketi configuration volume and path that should be /etc/heketi, with a hostPath mount to our working dir /usr/share/heketi on each of our nodes.
+
+<3>  Set up a Heketi db volume and path to store the heketi.db.  Notice we are using a volume on our actual GlusterFS cluster to store this heketi.db (volume should be greater than 40Gb).
+
+<4>  Set up a Heketi keys volume and path to store the SSH keys used by the sshexec of Heketi. Notice we are using another hostPath mount type.
+
+
+***NOTE:*** We have defined our port for Heketi to be 8888, default is 8080.
+
+#### Validate the Heketi pod is running.
+
+```
+	# kubectl get rc
+	NAME      DESIRED   CURRENT   READY     AGE
+	heketi    1         1         1         8m
+
+	# kubectl get pod -o wide
+	NAME           READY     STATUS    RESTARTS   AGE       IP              NODE
+	heketi-x71v9   1/1       Running   0          1h        192.168.1.209   k8dev3.rhs
+
+
+	# kubectl logs heketi-x71v9
+	Heketi v4.0.0-46-g8e58761
+	[heketi] INFO 2017/03/21 16:18:54 Loaded ssh executor
+	[heketi] INFO 2017/03/21 16:18:54 Loaded simple allocator
+	[heketi] INFO 2017/03/21 16:18:54 GlusterFS Application Loaded
+	Listening on port 8888
+```
+
+***NOTE:*** Because we used HOSTNAME_OVERRIDE environment variable to start local-up-cluster.sh and set the hostNetwork: true in our container
+the container picks up the actual ipaddr of the node it is running on.
+
+#### Test connection to Heketi.
+
+```
+
+	# curl http://k8dev3.rhs:8888/hello
+	Hello from Heketi
+```
+
+#### Set an environment variable for the Heketi Server.
+In a previous step we have installed the Heketi-Client on our 
+master node.
+
+
+```
+	# export HEKETI_CLI_SERVER=http://k8dev3.rhs:8888
+```
+
+### Using Heketi with Gluster
+
+Topology is used to tell Heketi about the environment, what nodes and devices it will manage.
```

Comment:
This sentence is clunky, please reword. Start with something like "A topology file" instead.